gulp-tasks/documentation.js

```diff
@@ -54,6 +54,11 @@ const documentPackage = (projectPath) => {
 
     handleBarsData.hasRefDocs = (projectMetadata.name !== 'sw-cli');
 
+    // sw-cli needs a custom install command
+    if (projectMetadata.name === 'sw-cli') {
+      handleBarsData.installCmd = `npm install -g sw-cli`;
+    }
+
```

Comment:
Had to do a double take on this, but totally makes sense given this is a global CLI you're going to use differently to more local modules.